closed/src/java.base/share/classes/openj9/internal/security/RestrictedSecurity.java

```diff
@@ -137,6 +139,14 @@ public String[] run() {
         }
         isOpenJCEPlusSupported = isOsSupported && isArchSupported;
 
+        // Check whether the OpenJCEPlus module exists.
+        isOpenJCEPlusModuleExist = false;
+        ModuleLayer layer = ModuleLayer.boot();
+        Optional<Module> module = layer.findModule("openjceplus");
+        if (module.isPresent()) {
+            isOpenJCEPlusModuleExist = true;
+        }
```

Comment:
The field should be `final` and initialized as:
```java
        isOpenJCEPlusModuleExist = module.isPresent();
```

Reply:
Done.